ract/cli.py

```diff
@@ -24,7 +24,10 @@
 @click.option(
     "--seconds", default=10.,
     help="Number of seconds to run for. Default is 10.")
-def ract(cube, art, fps, seconds):
+@click.option(
+    "--disp_timer", default=False,
```

Comment:
This should be `--disp-timer` probably (click will convert the Python argument name to `disp_timer`).